src/Loop.php

```diff
@@ -59,155 +59,162 @@ public static function stop()
     /**
      * Defer the execution of a callback.
      *
-     * @param callable $callback The callback to defer.
+     * @param callable(mixed $data, string $watcherIdentifier) $callback The callback to defer.
```

Comment:
> I included them for consistency

+1

> If the $watcherIdentifier param is removed from these callbacks then, for consistency, would it not make sense for $data always to be the first param passed to callbacks, as proposed by @trowski?

+1

Consistency FTW. Don't care which, but would like to see it to always be the same whichever is eventually chosen.